lib/recommender_system/irt_recommender.rb

```diff
@@ -0,0 +1,28 @@
+module RecommenderSystem
+  class IrtRecommender < RecommenderSystem::Recommender
+    def get_probability
+      los = self.learning_objects
+      list = Hash.new
+      los.each do |lo|
+        irt_values = lo.irt_values.first
```

Comment:
Vzdy sa vezme prva `irt_value` daneho LO (learning objectu). Moze mat LO viac zaznamov `irt_value`? Lebo v DB je vztah `1:N`. Ak je to naozaj vzdy `1:1` a neevidujeme ziadnu historiu (zatial zrejme nie, lebo sa nepouziva posledny podla datumu a pod., ale prvy a vlastne tabulka `irt_values` ani nema timestamps), tak potom je `difficulty` a `discrimination` predsa vlastnostou priamo LO a mozu to byt stlpce v jeho tabulke...